exetera/core/fields.py

```diff
@@ -288,6 +320,13 @@ def complete(self):
 
 class ReadOnlyIndexedFieldArray:
     def __init__(self, field, indices, values):
+        """
+        Construct a IndexFieldArray which used as the wrapper of data in IndexedStringField.
```

Comment:
Construct a ReadOnlyIndexedFieldArray instance. This class is an implementation detail, used to access data in indexed fields. As such, instances of ReadOnlyIndexedFieldArray should only be created by the Fields themselves, and not by users of ExeTera.